asreview/analysis/analysis.py

```diff
@@ -276,7 +276,7 @@ def rrf(self, val=10, x_format="percentage", **kwargs):
         return (None, None, None)
 
     def avg_time_to_discovery(self, result_format="number"):
-        """Get the best/last estimate on how long it takes to find a paper.
+        """Get the best/last estimate on how long it takes to find a paper, the Time to Discovery (TD).
```

Comment:
```
        """Compute the average Time to Discovery.

        Get the best/last estimate on how long it takes to find a paper, the Time 
        to Discovery (TD).

        Returns
        -------
```

The first line should be short (72char). Longer description below.

Reply:
Thanks for the heads up!

Reply:
Like this? 
>      """Get the best/last estimate on how long it takes to find a paper.
>         Known as Time to Discovery (TD).

Reply:
There should be another newline after the first line. Like the format above. 

1. one line description
2. blank line
3. Long description.

Reply:
I have already changed it ;). Though the long description could still use some work.

Reply:
Thanks